docs/Commands.md

```diff
@@ -19,3 +19,20 @@ Add Command `Hello chat :)` with the trigger `/hello`. Now typing `/hello` in ch
   Example: Add Command `Have a {1+} day!` with trigger `/day`. Now typing `/day very super nice` will send `Have a very super nice day!`
 
 - You can use `{{1}` if you want to send `{1}` literally.
+
+Placeholders below are available in nightly:
```

Comment:
I feel like this should be changed to something that's along what was already established in https://github.com/Chatterino/wiki/pull/69/files#diff-e721b9868c8be65cc8830d44796b2ac3cd6838890e8c79926b0ef38dc46f7095R14
basically just make `nightly` something like `[nightly release](../Help/#what-is-nightly-and-how-to-use-install-it)`

Reply:
```suggestion

Placeholders below are available on nightly:
```
I thought about this when I originally looked over this PR, and I still believe this is more correct.
`available on nightly builds…`

also the above suggestion is good too